pkgs/os-specific/linux/zsa-udev-rules/default.nix

```diff
@@ -20,6 +20,8 @@ stdenv.mkDerivation rec {
     mkdir -p $out/lib/udev/rules.d
     cp dist/linux64/50-oryx.rules $out/lib/udev/rules.d/
     cp dist/linux64/50-wally.rules $out/lib/udev/rules.d/
+    echo '# Voyager' >> $out/lib/udev/rules.d/50-wally.rules
```

Comment:
as the voyager is not flashable via wally, i would suggest to drop a 50-keymapp.rules in there or so.

Reply:
A quick question: have they confirmed that it is not flashable via Wally? I know that in the documentation they only give rules for the Voyager with Keymapp, but that can be an oversight.

Reply:
> drop a 50-keymapp.rules in there or so

@Shawn8901 I'd love to incorporate your suggestion since you seem to know much more about this than I do (I just copied the rule from ZSA's wiki). However, I don't know how. I've opened an identical [PR](https://github.com/zsa/wally/pull/135) in ZSA's repo and if you wish to `add a suggestion` I'll gladly accept it.

Reply:
> A quick question: have they confirmed that it is not flashable via Wally? I know that in the documentation they only give rules for the Voyager with Keymapp, but that can be an oversight. Update: I've tested it and I couldn't get it to work.

I have a voyager, and at least wally-cli does not know the USB IDs of the voyager. I guess it might be flashable if someone adds the correct IDs, but I don't want to test if, as I don't know how I might cover if I break it.
I can test tomorrow if wally is able to flash in case I get it running (we just have wally-cli in nixpkgs afaik)

Reply:
> > drop a 50-keymapp.rules in there or so
> 
> @Shawn8901 I'd love to incorporate your suggestion since you seem to know much more about this than I do (I just copied the rule from ZSA's wiki). However, I don't know how. I've opened an identical [PR](https://github.com/zsa/wally/pull/135) in ZSA's repo and if you wish to `add a suggestion` I'll gladly accept it.

I honestly don't know to much about that, I have just from overall design upstream has the onyx file, which makes online training via chrome working and the wally file, that makes flashing via wally working. So it feels natural to me, to create a keymapp file for those that is specific for keymapp. 
For my voyager at the moment I have just dropped the rules in udev.extraRules as I did not yet have.preference how to upstream (neither to nixpkgs or wally repo)